packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -121,17 +123,24 @@ export class UnfurlService {
             await page.setRequestInterception(true);
             page.on('request', (request: any) => {
                 const requestUrl = request.url();
-                if (blockedUrls.includes(requestUrl)) {
+                const urlHost = url.split('/')[2];
+                // Only allow request to the same host
+                if (!requestUrl.includes(urlHost)) {
                     request.abort();
                     return;
                 }
-
                 request.continue();
             });
-            await page.goto(url, {
-                timeout: 180000,
-                waitUntil: 'networkidle0',
-            });
+            try {
+                await page.goto(url, {
+                    timeout: 60000,
+                    waitUntil: 'networkidle0',
+                });
+            } catch (e) {
```

Comment:
instead of waiting 3 mins,and timeout. 

Now we wait 1 minute and return whatever is available in the page. This should ensure we always return an image. 

Worst case scenario: Some charts will not display (but they were not going to return anyway...) 
![image](https://user-images.githubusercontent.com/1983672/221130750-ee7aca79-f88c-467a-9302-df6ead16e810.png)